modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -2986,6 +2993,12 @@ public CodegenProperty fromProperty(String name, Schema p) {
         } else if (ModelUtils.isFreeFormObject(p)) {
             property.isFreeFormObject = true;
         } else if (ModelUtils.isAnyTypeSchema(p)) {
+            // The 'null' value is allowed when the OAS schema is 'any type'.
+            // See https://github.com/OAI/OpenAPI-Specification/issues/1389
+            if (Boolean.FALSE.equals(p.getNullable())) {
+                LOGGER.error("Schema '{}' is any type, which includes the 'null' value. 'nullable' cannot be set to 'false'", p.getName());
```

Comment:
I would use LOGGER.warn instead and make it clear it's for the upcoming OAS v3.1

(after OAS v3.1 release, I would expect the parser (swagger-parser) to catch this kind of issue)

Reply:
done